### ISSUE TITLE
Error message when downloadin all videos from TV4-play Premium 

### ISSUE THREAD

**Issue by qnorsten** [spaam/svtplay-dl#280]
Think you broke something when fixing the naming issue with the TV4-play Premium as I mentioned in a separate issue. 
Now when i try to download all the episodes of any series from the kids part of TV4-play premium I get an error message. 
Example Url: http://www.tv4play.se/barn/yu-gi-oh?video_id=1622568 
Command that i run: P:\Programing\svtplay-dl\lib\svtplay_dl\__main__.py -u <username> -p <password> - A http://www.tv4play.se/barn/yu-gi-oh?video_id=1622568 

Error message i get is 
```python
Traceback (most recent call last):
  File "P:\Programing\svtplay-dl\lib\svtplay_dl\__main__.py", line 13, in <modul
e>
    svtplay_dl.main()
  File "P:\Programing\svtplay-dl\lib\svtplay_dl\__init__.py", line 343, in main
    get_media(url, options)
  File "P:\Programing\svtplay-dl\lib\svtplay_dl\__init__.py", line 161, in get_m
edia
    episodes = stream.find_all_episodes(options)
  File "P:\Programing\svtplay-dl\lib\svtplay_dl\service\tv4play.py", line 163, i
n find_all_episodes
    show, video_id)
NameError: name 'show' is not defined

```

**Comment by Sopor-** [comment]
I get this when using -A on regular download on TV4
```
Traceback (most recent call last):
  File "/usr/local/bin/svtplay-dl", line 5, in <module> pkg_resources.run_script('svtplay-dl==0.20.2015.09.13', 'svtplay-dl')
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 528, in run_script self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1394, in run_script execfile(script_filename, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.09.13-py2.7.egg/EGG-INFO/scripts/svtplay-dl", line 7, in <module> svtplay_dl.main()
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.09.13-py2.7.egg/svtplay_dl/__init__.py", line 341, in main get_media(url, options)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.09.13-py2.7.egg/svtplay_dl/__init__.py", line 140, in get_media url, stream = generic.get(sites)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.09.13-py2.7.egg/svtplay_dl/service/__init__.py", line 150, in get return url, stream
UnboundLocalError: local variable 'url' referenced before assignment
```

**Comment by Sopor-** [comment]
Above was not latest svtplay-dl master. Here is a new one:
```
$ svtplay-dl -A http://www.tv4play.se/program/solsidan
Traceback (most recent call last):
  File "/usr/local/bin/svtplay-dl", line 5, in <module> pkg_resources.run_script('svtplay-dl==0.20.2015.10.08', 'svtplay-dl')
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 528, in run_script self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1394, in run_script execfile(script_filename, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.10.08-py2.7.egg/EGG-INFO/scripts/svtplay-dl", line 7, in <module> svtplay_dl.main()
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.10.08-py2.7.egg/svtplay_dl/__init__.py", line 343, in main get_media(url, options)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.10.08-py2.7.egg/svtplay_dl/__init__.py", line 161, in get_media episodes = stream.find_all_episodes(options)
  File "/usr/local/lib/python2.7/dist-packages/svtplay_dl-0.20.2015.10.08-py2.7.egg/svtplay_dl/service/tv4play.py", line 163, in find_all_episodes show, video_id)
NameError: global name 'show' is not defined
```

**Comment by Sopor-** [comment]
I also noticed that #251 is back again but it seems to be for single episode download from TV4 only.

**Comment by Sopor-** [comment]
Will this fix #251 and #259 too or should i create new issues for those?

**Comment by spaam** [comment]
if you want it fixed yes.  i need to have python version, version of requests, OS and url.
if i dont get that. i can't fix it. and patch welcome
